build.gradle

```diff
@@ -249,6 +262,7 @@ project('client') {
         // The client fallsback to NIO based transport if epoll is not available.
         compile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion, classifier: 'linux-x86_64'
         compile group: 'com.google.guava', name: 'guava', version: guavaVersion
+        testCompile project(path: ':shared:security', configuration: 'testOutput')
```

Comment:
Why was this needed?

Reply:
this line is needed as project client has test dependency over project security, when this is run gradle will run the jarTest task in the security project in the build.gradle.